source/exec.tex

```diff
@@ -5385,17 +5411,18 @@
     void @\exposid{abandon}@() noexcept;                                                // \expos
 
   private:
-    using @\exposid{alloc-t}@ =                                                         // \expos
-      allocator_traits<Alloc>::template rebind_alloc<@\exposid{spawn-future-state}@>;
-
-    @\exposid{alloc-t}@ @\exposid{alloc}@;                                                          // \expos
+    Alloc @\exposid{alloc}@;                                                            // \expos
     @\exposid{ssource-t}@ @\exposid{ssource}@;                                                      // \expos
     @\exposid{op-t}@ @\exposid{op}@;                                                                // \expos
     Token @\exposid{token}@;                                                            // \expos
     bool @\exposid{associated}@;                                                        // \expos
 
     void @\exposid{destroy}@() noexcept;                                                // \expos
   };
+
+  template<class Alloc, @\libconcept{scope_token}@ Token, @\libconcept{sender}@ Sender, class Env>        // \expos
```

Comment:
Do we need the \expos comment here? It's not in the paper.

Where is this deduction guide used, only in the spec, or is its existence observable to users?

Reply:
I've raised it on mattermost